gbus/abstractions.go

```diff
@@ -24,6 +24,7 @@ const (
 type BusConfiguration struct {
 	MaxRetryCount     uint
 	BaseRetryDuration int
+	Serializer        Serializer
```

Comment:
@adiweiss we already have the WithSerializer on the bus interface.
no need to add this here as well